### DISCUSSION TITLE
Start RustDesk client on Windows startup

### DISCUSSION THREAD

**Discussion by OverHash** [rustdesk/rustdesk#5113] [General]
I believe this feature existed in RustDesk v1.1.9, but no longer exists. Before, RustDesk would register a service under `services.msc` which would allow the RustDesk client to start when a Windows machine boots.

This feature seems to no longer exist in the latest RustDesk version -- it would be nice to have this functionality.

#3930 Is similar to this discussion post -- but is targeted for the self-hosted server.
#1761 suggests a way to do this -- but this is not a native solution and was under the "General" category.

**Comment by rustdesk** [comment]
Not working? It works well for me and most users.
<img width="354" alt="image" src="https://github.com/rustdesk/rustdesk/assets/71636191/0b70cfbc-9ed4-4074-a0d4-b565a3ea5c0e">

<img width="248" alt="image" src="https://github.com/rustdesk/rustdesk/assets/71636191/ad87da03-29db-4c8f-b183-13327dbf2e53">

**Reply by OverHash** [reply]
Hmm, at what step does RustDesk install the service? For me the RustDesk service does not exist -- but it did before in v1.1.9.

**Reply by OverHash** [reply]
Solved by installing RustDesk (see the UAC prompt at the bottom right).

**Reply by jvschiavo** [reply]
Would this work if installed via chocolatey?

**Comment by andy-page-developer** [comment]
Same issue for me.

**Reply by OverHash** [reply]
Hey, I solved the issue by installing RustDesk rather than running it in portable mode (see the UAC prompt at the bottom right if you have it).

**Comment by andy-page-developer** [comment]
Oh my , how did I not see the install button.

Thank you.
